src/components/Audiences/index.js

```diff
@@ -22,7 +22,9 @@ const createAudiences = ({ logger }) => {
     lifecycle: {
       onResponse({ response }) {
         const destinations = response.getPayloadsByType("activation:push");
-        return processDestinations(destinations);
+        // we're now returning an object at onResponse
+        // here we need to add 'then(noop)' because we are not returning a value
+        return processDestinations(destinations).then(noop);
```

Comment:
Hmm....why is the promise returned by processDestinations being resolved with a value of any kind? Rather than doing a `.then(noop)` here, this seems like something that should be fixed within `processDestinations`.

Reply:
yes, actually we don't need it here